Resources/public/js/ezrecommendation.js

```diff
@@ -0,0 +1,77 @@
+// register HandleBars helpers when library is loaded
```

Comment:
for the same reason, you don't define all the PHP classes in one file. Here you have a JavaScript files that defines 3 things:
1. the Handlebars helpers
2. the recommendation client
3. the recommendation rendered

it's 3 very different things => 3 files.